pandas/tools/merge.py

```diff
@@ -266,16 +266,23 @@ def merge_asof(left, right, on=None,
                by=None, left_by=None, right_by=None,
                suffixes=('_x', '_y'),
                tolerance=None,
-               allow_exact_matches=True):
+               allow_exact_matches=True,
+               direction='backward'):
     """Perform an asof merge. This is similar to a left-join except that we
     match on nearest key rather than equal keys.
 
-    For each row in the left DataFrame, we select the last row in the right
-    DataFrame whose 'on' key is less than or equal to the left's key. Both
-    DataFrames must be sorted by the key.
+    Both DataFrames must be sorted by the key.
 
-    Optionally match on equivalent keys with 'by' before searching for nearest
-    match with 'on'.
+    For each row in the left DataFrame, a "backward" search selects the last
+    row in the right DataFrame whose 'on' key is less than or equal to the
+    left's key. A "forward" search selects the first row in the right DataFrame
```

Comment:
can you make this a bulleted list (for forward, backward, nearest), easier to read